scripts/publish-new.ts

```diff
@@ -63,6 +63,7 @@ async function main() {
 		'lerna.json',
 		...packageJsonFilesToAdd,
 		BUBLIC_ROOT + '/packages/*/src/version.ts',
+		BUBLIC_ROOT + '/packages/*/src/lib/ui/version.ts',
```

Comment:
I think it should be safe to do this, just in case we move those files around at some point.

```suggestion
		BUBLIC_ROOT + '/packages/*/src/**/version.ts',
```